os/notes-for-distributors.md

```diff
@@ -18,9 +18,30 @@ gpg --verify coreos_production_openstack_image.img.bz2.sig
 
 ## Image customization
 
-Customizing a Container Linux image for a specific operating environment is easily done through [cloud-config](https://github.com/coreos/coreos-cloudinit/blob/master/Documentation/cloud-config.md), a YAML-based configuration standard that is widely supported. As a provider, you must ensure that your platform makes this data available to Container Linux, where it will be parsed during the boot process.
+There are two predominant ways that a Container Linux image can be easily customized for a specific operating environment: through Ignition, a first-boot provisioning tool that runs during a machine's boot process, and through [cloud-config](https://github.com/coreos/coreos-cloudinit/blob/master/Documentation/cloud-config.md), an older tool that runs every time a machine boots.
 
-Use cloud-config to handle platform specific configuration such as custom networking, running an agent on the machine or injecting files onto disk. Container Linux will automatically parse and execute `/usr/share/oem/cloud-config.yml` if it exists. Your cloud-config should create additional units that process user-provided metadata, as described below.
+### Ignition
+
+[Ignition][ignition] is a tool that acquires a JSON config file when a machine first boots, and uses this config to perform tasks such as formatting disks, creating files, modifying and creating users, and adding systemd units. How Ignition acquires this config file varies per-platform, and it is highly recommended that providers ensure Ignition has [support for their platform][ign-platforms].
+
+Ignition has a default config for each supported platform that is appended to any user provided config. Use Ignition to handle platform specific configuration such as custom networking, running an agent on the machine, or injecting files onto disk.
```

Comment:
This first statement is an implementation detail and will be removed eventually.